scripts/build/babel.config.js

```diff
@@ -1,15 +1,15 @@
 // Babel builder doesn't generate CommonJS, but ES modules.
 // Rollup, however, needs `modules: false, babel modules: true`
-module.exports = function(options) {
+module.exports = function(commonJs) {
   const presets = [
     [
       "@babel/preset-env",
       {
         targets: {
-          node: 8,
-          browsers: "> 1%, last 2 versions"
+          node: 14,
+          browsers: "> 1%, last 2 versions, not dead"
```

Comment:
`not dead` is recommended by browserlist itself. Otherwise, query `last 2 versions` will pull internet explorer and other dead browsers.